crates/cheatcodes/src/inspector.rs

```diff
@@ -1354,7 +1354,13 @@ impl Cheatcodes {
                         // inside it.
                         self.gas_metering_create = Some(None)
                     }
-                    opcode::STOP | opcode::RETURN | opcode::SELFDESTRUCT | opcode::REVERT => {
+                    opcode::STOP => {
```

Comment:
dont know if this is right, stop and return are effectively the same

Reply:
yeah, but this is to accommodate the edge case when `vm.pauseGasMetering();` is the last call in test as described in https://github.com/foundry-rs/foundry/issues/5491 so can be STOP only

Reply:
can you add a case with assembly { return(0, 0) }

Reply:
sure, do you mean a tests like
```Solidity
    function testWithAssembly() public {
        vm.pauseGasMetering();
        assembly {
            return(0, 0)
        }
    }
```
which returns
```[PASS] testWithAssembly() (gas: 3083)``` or you thinking to something different?